modules/rosa-updating-rosa-cli.adoc

```diff
@@ -0,0 +1,60 @@
+:_content-type: PROCEDURE
+[id="rosa-updating-the-rosa-cli_{context}"]
+= Updating the ROSA CLI
+
+Update to the latest compatible version of the {product-title} CLI.
+
+.Procedure
+
+. Confirm that a new version of the ROSA CLI is available.
++
+[source,terminal]
+----
+$ rosa version
+----
++
+.Example
+[source,terminal]
+----
+$ rosa version
+1.2.12
+There is a newer release version '1.2.15', please consider updating: https://mirror.openshift.com/pub/openshift-v4/clients/rosa/latest/
+----
+
+. Download the latest compatible version of the ROSA CLI.
```

Comment:
Can you end all of these steps with colons?

Reply:
Done, thankyou!